example/Makefile.inc

```diff
@@ -1,6 +1,6 @@
 include $(top_srcdir)/platform/@with_platform@/Makefile.inc
 LIB   = $(top_builddir)/lib
-LDADD = $(LIB)/libodp-linux.la $(LIB)/libodphelper.la
+LDADD = $(LIB)/libodp-linux.la $(LIB)/libodphelper.la $(DPDK_PMDS)
```

Comment:
checked with make V=1 and saw it worked, although in automake manual it outlines that:

> https://www.gnu.org/software/automake/manual/automake.html#Libtool-Convenience-Libraries
> LDADD to do so. This variable is used to specify additional objects or libraries to link with; it is inappropriate for specifying specific linker flags, you should use AM_LDFLAGS for this purpose.

the DPDK_PMDS expands to -Wl,--whole-archives,....., which seems belong to the LDFLAGS? but since it already works it is OK to add in LDADD.